core/queue/serializers.py

```diff
@@ -1,9 +0,0 @@
-from rest_framework import serializers
```

Comment:
Why was this file deleted?

Reply:
this serializer was never used.  The queue just uses the already existing Program serializer , since there is no queue table.